### DISCUSSION TITLE
how to enable caps lock?

### DISCUSSION THREAD

**Discussion by vekexasia** [manna-harbour/miryoku#327] [Q&A] [answered]
I've tried keeping the two shifts pressed but it looks like it is doing something else,,,

**Comment by vekexasia** [comment]
for those landing here... the nav layer has acaps word functionality

**Comment by manna-harbour** [comment, marked as answer]
Caps Lock is on [Nav](https://github.com/manna-harbour/miryoku/tree/master/docs/reference#nav), but in Miryoku ZMK [Caps word](https://zmk.dev/docs/behaviors/caps-word) is used in place of Caps Lock. Combine with Shift for Caps Lock. See https://github.com/manna-harbour/miryoku_zmk#caps-word.
